android/src/paper/java/com/swmansion/gesturehandler/ReactContextExtensions.kt

```diff
@@ -0,0 +1,9 @@
+package com.swmansion.gesturehandler
+
+import com.facebook.react.bridge.ReactContext
+import com.facebook.react.uimanager.UIManagerModule
+import com.facebook.react.uimanager.events.Event
+
+fun ReactContext.dispatchEvent(event: Event<*>) {
+    this.getNativeModule(UIManagerModule::class.java)!!.eventDispatcher.dispatchEvent(event)
```

Comment:
Shouldn't `let` be used here to be `null-safe`?

Reply:
Should we be concerned about that? `UIManagerModule` shouldn't be `null` if everything is working correctly, no? But if that's the case, wouldn't `?.` work as well?

Reply:
I have no knowledge if it should be present there, but I think it should be handled in a better way than `!!`, by e.g. throwing an exception with descriptive message or something like that.

Reply:
I wrapped it with a try-catch block in https://github.com/software-mansion/react-native-gesture-handler/pull/1893/commits/589e641049566b13c7aafc72d9a7fda4bec0306d.